ownCloud/Bookmarks/BookmarkViewController.swift

```diff
@@ -810,6 +862,23 @@ class BookmarkViewController: StaticTableViewController {
 		}
 	}
 
+	func updateStorageInfo() {
+		if bookmark != nil {
+			if let vaultURL = OCVault(bookmark: bookmark!).filesRootURL {
+				FileManager.default.calculateDirectorySize(at: vaultURL) { (size) in
+					if size != nil {
+						let occupiedSpace = self.byteCounterFormatter.string(fromByteCount: size!)
+						self.offlineStorageInfoRow?.value = occupiedSpace
+					}
```

Comment:
Error handling is missing for `size == nil`… maybe could show "not available" or "n/a" in that case.

Reply:
@felix-schwarz right now it shows "unknown" from the beginning, since the calculation is done on a background thread and takes a couple of seconds. So if size is nil, it will just stay "unknown".

Reply:
Missed that. Thanks!